ihmc-perception/src/main/java/us/ihmc/perception/gpuHeightMap/RapidHeightMapExtractor.java

```diff
@@ -793,17 +793,14 @@ public Mat getCroppedImage(Point3DReadOnly origin, int globalCenterIndex, Mat im
       return imageToCrop.apply(cropWindowRectangle);
    }
 
-   public static HeightMapData packHeightMapData(RapidHeightMapExtractorInterface heightMapExtractor, HeightMapData heightMapDataToPack)
+   public static HeightMapData packHeightMapData(RapidHeightMapExtractorInterface heightMapExtractor)
```

Comment:
Cleanup method

Reply:
I think it should be `getHeightMapData()` since it's not packing it anymore. To be honest I'm even more confused why it's a static method. Can't we make it a default method of the interface?